examples/assets/config/layer/workspace_tileset.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
we should create an issue to get remind to change this file when the workspace backend will be ready